vivisect/parsers/blob.py

```diff
@@ -10,6 +10,12 @@
     'arm':'armcall',
     }
 
+def isParser(bytez):
```

Comment:
"taste" ?

`isParser` makes me think that we check the `bytez` to see if the bytez are a parser, versus can be parsed by a given parser.

Reply:
suggestion?  i think of how it will be called "module.isParser(bytez)" and it makes sense.  but you have a good point as well.